src/home/CallList.tsx

```diff
@@ -9,7 +9,10 @@ import { Link } from "react-router-dom";
 import { MatrixClient } from "matrix-js-sdk/src/client";
 import { RoomMember } from "matrix-js-sdk/src/models/room-member";
 import { Room } from "matrix-js-sdk/src/models/room";
-import { FC } from "react";
+import { FC, useCallback, MouseEvent } from "react";
+import { t } from "i18next";
```

Comment:
lets use the `const {t} = useTranslation()` hook here instead to have it reactive.

Reply:
Ah, I fell for the code completion in my editor there. 🤦‍♂️